src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Shapes/Given_Path.cs

```diff
@@ -41,5 +45,29 @@ public void Should_Not_Include_Control_Points_Bounds()
 			Assert.IsTrue(Math.Abs(50 - SUT.DesiredSize.Height) <= 1, $"Actual size: {SUT.DesiredSize}");
 #endif
 		}
+
+		// This tests a workaround for a Linux-specific skia bug.
+		// This doesn't actually repro for some reason, so it's here for documentation purposes.
```

Comment:
The test passes on Linux even without this PR. RenderTargetBitmap renders on a new surface, which seems to prevent the problem from appearing. The problem only appears with the renderer's SKSurface.

Reply:
In this case I would suggest creating a sample page which illustrates the problem and is marked as manual test for QA to verify against regressions